nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/actor/AbstractClusterListener.scala

```diff
@@ -167,7 +174,7 @@ abstract class AbstractClusterListener extends Actor with ActorLogging with Futu
 
           val locationsToAdd: Seq[LocationWithCoordinates] = retrieveLocationsToAdd.diff(outdatedLocations.locations)
 
-          log.debug(s"locations to add from node $nodeId $locationsToAdd")
+          log.info(s"locations to add from node $nodeId $locationsToAdd")
```

Comment:
maybe a colon and a tab help this string readability `"locations to add from node $nodeId: \t$locationsToAdd"`